server/models/Collective.js

```diff
@@ -2979,6 +2979,14 @@ export default function (Sequelize, DataTypes) {
         t => !t.platformFeeInHostCurrency && t.hostFeeInHostCurrency,
       ),
     );
+    const pendingHostFeeShare = Math.abs(
+      sumByWhen(
+        transactions,
+        t => round((t.hostFeeInHostCurrency * (t.data?.hostFeeSharePercent || plan.hostFeeSharePercent)) / 100),
+        t => !t.platformFeeInHostCurrency && t.hostFeeInHostCurrency,
+        isPendingTransaction,
```

Comment:
`sumByWhen` accepts only three arguments:
```suggestion
        t => !t.platformFeeInHostCurrency && t.hostFeeInHostCurrency && isPendingTransaction(t)
```